pkg/metrics/metrics.jsx

```diff
@@ -535,6 +593,56 @@ class CurrentMetrics extends React.Component {
         this.setState(newState);
     }
 
+    cgroupRow = (name, value, is_user, is_container, uid, valueB) => {
+        // valueB: optional parameter, used for cgroup disk write
+        const getCachedPodName = (uid, containerid) => this.state.podNameMapping[uid] && this.state.podNameMapping[uid][containerid];
+        const podman_installed = cockpit.manifests && cockpit.manifests.podman;
+
+        function cgroupClickHandler(name, isUser, isContainer, uid) {
+            if (isContainer) {
+                const containerName = getCachedPodName(uid, name);
+                if (containerName) {
+                    cockpit.jump("/podman#/?name=" + containerName);
+                } else {
+                    cockpit.jump("/podman");
+                }
+            } else {
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17692-20220913-080738-76591a4a-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)